lib/tokens.ts

```diff
@@ -30,14 +24,6 @@ export async function enrichTokenMetadata(
   tokenAddress: PublicKey,
   rpcProvider: Provider
 ): Promise<TokenProps & { isFallback?: boolean }> {
-  // first, is it USDC? we handle that manually with no fetching
-  if (
```

Comment:
@Dodecahedr0x  @R-K-H what are your thoughts here? Without this, USDC will not display I believe. I don't think there is metadata for that token?

Reply:
So we can fix the mUSDC, I cannot fix the devnet USDC. HOWEVER there still is token data, but likely pulled through some old spl-token library to support what you can get from https://faucet.circle.com/

https://solscan.io/token/4zMMC9srt5Ri5X14GAgXhaHii3GnPAEERYPJgZJDncDU shows a logo... Not exactly sure how they fetch that...

Reply:
I THINK it may have been this https://github.com/solana-labs/token-list

Reply:
Oh yea this thing `@solana/spl-token-registry`

Reply:
I believe that images should not be stored in the futarchy TS SDK, but rather in a futarchy React (or FE) SDK. In the mean time, I guess the FE can handle it directly

Reply:
@R-K-H do you think we should implement this in the SDK then?
https://www.npmjs.com/package/@solana/spl-token-registry